dockstore-integration-testing/src/test/java/io/dockstore/webservice/metrics/MetricsDataS3ClientIT.java

```diff
@@ -78,31 +82,37 @@
 
 @ExtendWith(LocalstackDockerExtension.class)
 @Tag(LocalStackTest.NAME)
-@LocalstackDockerProperties(imageTag = LOCALSTACK_IMAGE_TAG, services = { ServiceName.S3 }, environmentVariableProvider = MetricsDataS3ClientIT.LocalStackEnvironmentVariables.class)
-public class MetricsDataS3ClientIT extends BaseIT {
-    public static final String LOCALSTACK_IMAGE_TAG = "1.3.1";
+@LocalstackDockerProperties(imageTag = IMAGE_TAG, services = { ServiceName.S3 }, environmentVariableProvider = LocalStackEnvironmentVariables.class)
+public class MetricsDataS3ClientIT {
     private static final Gson GSON = new Gson();
+    public static final DropwizardTestSupport<DockstoreWebserviceConfiguration> SUPPORT = new DropwizardTestSupport<>(
+            DockstoreWebserviceApplication.class, CommonTestUtilities.CONFIDENTIAL_CONFIG_PATH);
+    private static TestingPostgres testingPostgres;
     private static String bucketName;
+    private static String s3EndpointOverride;
     private static S3Client s3Client;
     private static MetricsDataS3Client metricsDataClient;
 
     @BeforeAll
-    public static void setup() throws URISyntaxException {
+    public static void setup() throws Exception {
+        CommonTestUtilities.dropAndRecreateNoTestData(SUPPORT);
```

Comment:
I think I need this so that `SUPPORT` is initialized and the webservice application is started before I can initialize the other fields using `SUPPORT.getConfiguration()`